internal/vue-sandbox/src/local.ts

```diff
@@ -1,5 +1,5 @@
 import "@fkui/theme-default";
-import "@fkui/design";
+import "@fkui/design/src/fkui.scss";
```

Comment:
Borde vi skapa en `local.scss` så bör importen lira?

Reply:
Den tidigare fungerade med. Jag gjorde motsvarande ändring som är i `local.ts` för Vue-paketet, den gör att en kodändring i designpaketet slår igenom direkt (tar man in en css så måste man bygga om designpaketet).

Reply:
Samma gäller för `@fkui/theme-default` också.

Reply:
Fixat för alla local.ts